splash/resources.py

```diff
@@ -38,7 +39,7 @@ def render(self, request):
             return Resource.render(self, request)
         except BadOption as e:
             request.setResponseCode(400)
-            return str(e) + "\n"
+            return str(e).encode('utf-8') + b"\n"
```

Comment:
What's most elegant way to do this? I'm thinking of just leaving out the encode part on python 2 since str(e) is essentially just bytes.